pkg/services/ngalert/models/alert_rule.go

```diff
@@ -143,6 +151,13 @@ var (
 		PanelIDAnnotation:                   {},
 		alertingModels.ImageTokenAnnotation: {},
 	}
+
+	// LabelsUserCannotSpecify are labels that the user cannot specify when creating an alert rule.
+	LabelsUserCannotSpecify = map[string]any{
```

Comment:
I deliberately used a name that does not mention system-owned because it contains the subset of system-owned, and including all would have a bigger impact, and therefore should be rolled out more carefully